bounded_counter/count_min_sketch.py

```diff
@@ -136,10 +142,10 @@ def log_encode(self, input):
         return log_value_base + remainder
 
     def decode(self, value):
-        if self.algorithm == algorithm_logcounter:
-            return self.log_decode(value)
-        else:
-            return value
+        return value
+
+    def __len__(self):
+        return int(self.distinct_counter.cardinality())
```

Comment:
Is this only approximative, right?

If so, there has to be a big fat warning (or perhaps even a differently named method, Python has a clear contract for `__len__` and things may break in unexpected places).

Reply:
Sure, let's do just "cardinality"!